package.json

```diff
@@ -17,6 +17,7 @@
     "lint:all": "eslint .",
     "lint": "pnpm lint:all",
     "lint:prod": "set NODE_ENV=production&& eslint .",
+    "lint:audit": "pnpm exec eslint-config-inspector",
```

Comment:
Helpful command to see what our ESLint Config actually generates for debugging purposes!